include/openssl/aes.h

```diff
@@ -67,6 +67,8 @@ extern "C" {
 
 #define AES_BLOCK_SIZE 16
 
+#define AES_GCM_NONCE_LENGTH 12
```

Comment:
```suggestion
#define FIPS_AES_GCM_NONCE_LENGTH 12
```
and define it in `aead.h`. Or `#define EVP_AEAD_AES_GCM_NONCE_LENGTH 12`

The nonce of AES-GCM doesn't have to be 12 bytes long.

Reply:
This nonce has to be 12 bytes...